app/assets/iframe/flex.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>WorldPay DDC Flex 3DS</title>
```

Comment:
The only people who are going to see this title are people are nosing around how our stuff works, so we may as well make this more informative for them: “Device data collection for Worldpay 3DS Flex”.